### PR TITLE
add note about HTTP/2 support and setup

### PR DESCRIPTION
**Pull request by eikenb**
Had the idea to put the HTTP/2 support  in a call-out note box. Wanted to see it in a live context.

### CODE REVIEW

File: website/content/docs/nia/configuration.mdx [hunk 1]
```diff
@@ -42,6 +42,8 @@ buffer_period {
 
 The `consul` block is used to configure Consul-Terraform-Sync connection with a Consul agent to perform queries to the Consul Catalog and Consul KV pertaining to task execution.
 
+-> **Note:** For improved performance of CTS communicating with the local Consul process you can use HTTP/2. To use HTTP/2 your local Consul needs to have TLS/HTTPS enabled with `tls.enabled = true`. The other relevant settings are `address = 8501` to use the (default) HTTPS port and `tls.verify = false` if using self-signed certificates for Consul. This can solve issues hitting Consul's maximum connections limits as well as just generally perform faster when processing lots of blocking queries.
```
Comment (findkim) [inline website/content/docs/nia/configuration.mdx:45]: The location of the callout is 💯.

A few suggestions:
* Clearer distinction between Consul agent config and CTS configs. For example
     * "The other relevant settings" -> "To configure Consul-Terraform-Sync to use HTTP/2, configure the Consul client to use the HTTPS port (`address = "example.consul.com:8501"`) and ...
* "perform faster" -> "perform more efficiently"

Comment (eikenb) [reply website/content/docs/nia/configuration.mdx:45]: Made the second change as suggested.

The first suggestion calls out that the sentence previous to it, isn't right. It is supposed to talk about CTS's consul configuration, but it sounds like it is talking about the configuration of consul itself. I'm going to fix that while trying to incorporate your suggestion.

Comment (findkim) [reply website/content/docs/nia/configuration.mdx:45]: I wasn't sure how to word the first one either so i prefixed it with "for example" 😅, glad that you get what I was trying to conve

Comment (eikenb) [reply website/content/docs/nia/configuration.mdx]: Reworded and pushed the update. Resolving this thread as outdated.

Comment (findkim) [reply website/content/docs/nia/configuration.mdx]: After seeing the update, I'm a confused now.

> To use HTTP/2 the local Consul needs to have [TLS/HTTPS configured](/docs/nia/configuration#tls) with, at least, the cert and key set. 
For my test setup I configured the local agent with the cert and key, and not the Consul block in CTS configuration that is now linked

Comment (findkim) [reply website/content/docs/nia/configuration.mdx]: After seeing the update, I'm a confused now.

> To use HTTP/2 the local Consul needs to have [TLS/HTTPS configured](/docs/nia/configuration#tls) with, at least, the cert and key set. 

For my test setup I configured the local agent with the cert and key, and not the Consul block in CTS configuration that is now linked. This reads that the cert and key is needed on the CTS consul block. Do you mind sharing your setup?

**Review by findkim — APPROVED** [review]

File: website/content/docs/nia/configuration.mdx (hunk 1 above)
Comment (eikenb) [reply website/content/docs/nia/configuration.mdx]: Wrong link. :facepalm: 
Pushing up fix.

**Review by findkim — APPROVED** [review]
New changes look even better!

**Pull request merged**